### PR TITLE
Improved hybrid test case

### LINKED ISSUE #130: Implement hybrid test case on the shortened data.

### PR DESCRIPTION
**Pull request by hechth**
Implemented improved version of the hybrid test case

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -110,4 +110,20 @@ compute_mz_sd <- function(feature_groups) {
     }
   }
   return(mz_sd)
+}
+
+#' @export
+get_num_workers <- function() {
```
Comment (xtrojak) [inline R/utils.R:116]: I suppose we can use this function also in other tests.

File: tests/testthat/test-hybrid.R [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+  expected <- arrow::read_parquet(
+    file.path(testdata, "hybrid", paste0(.test_name, "_recovered_feature_sample_table.parquet"))
+  )
 
   expect_equal(actual$recovered_feature_sample_table, expected)
-})
+}, patrick::cases(
+  mbr = list(
+    files = c("mbr_test0.mzml", "mbr_test1.mzml", "mbr_test2.mzml"),
+    ci_skip = TRUE
```
Comment (xtrojak) [inline tests/testthat/test-hybrid.R:24]: Why is this skipped? It's pretty small test case.

**Review by xtrojak — APPROVED** [review]
Nice improvement @hechth !

File: tests/testthat/test-hybrid.R (hunk 2 above)
Comment (hechth) [reply tests/testthat/test-hybrid.R:24]: It takes about 5 minutes to run or so so ... and we are running the other test case.

File: R/utils.R (hunk 1 above)
Comment (hechth) [reply R/utils.R:116]: yes - we can make an issue for it.

**Pull request merged**